### PR TITLE
Cleanup code style in Microsoft.CodeAnalysis.Features

### PR DESCRIPTION
**Pull request by sharwell**
Each commit either applies one Fix All in Project operation (credited to @dotnet-bot) or patches the previous Fix All operation with any observed corrections (credited to @sharwell).

### CODE REVIEW

File: src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ internal abstract class FixSomeCodeAction : CodeAction
         private static readonly HashSet<string> s_predefinedCodeFixProviderNames = GetPredefinedCodeFixProviderNames();
 
         internal readonly FixAllState FixAllState;
-        private readonly bool _showPreviewChangesDialog;
+        private bool _showPreviewChangesDialog;
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs]: Was tehre a bug in the 'make readonly' fixer that caused this?

**Comment by CyrusNajmabadi** [comment]
Could we also move to static-local-functions if possible?

File: src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs (hunk 1 above)
Comment (sharwell) [reply src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs]: Yes. The commit links to the bug.

**Comment by sharwell** [comment]
> Could we also move to static-local-functions if possible?

I'll take a look at this in the morning. I'm using this project to review the code fix behaviors looking for bugs so that will be a good one to check.

File: src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs]: Got it.  Thanks!

**Comment by sharwell** [comment]
@CyrusNajmabadi currently there is no refactoring available for that, at least not in Visual Studio 2019 RC 3.

**Comment by CyrusNajmabadi** [comment]
Hrmmm... i could have sworn i wrote it (but my brain is often filled with flights of fancy).  It's also possible it's in 16.1...

**Pull request merged**